wallet/connect/index.md

```diff
@@ -0,0 +1,55 @@
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------:|:------------------------------:|:---------------:|
+| Connect from a web dapp to the MetaMask extension                                                                                                                                                                                                                                                  |         ✓         |               ✓                |        ✓        |
+| Connect from a web dapp, desktop, mobile, and gaming dapps to MetaMask Mobile                                                                                                                                                                                                                      |         ✓         |               ✓                |                 |
+| Connect to MetaMask using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                          |         ✓         |               ✓                |        ✓        |
+| Connect to other wallets using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                     |                   |               ✓                |        ✓        |
+| Custom RPC methods such as [`connectAndSign`](../how-to/sign-data/connect-and-sign.md), [custom modals](../how-to/display/display-custom-modals.md), [read-only requests](../how-to/make-read-only-requests.md) using Infura API, and [RPC request batching](../how-to/batch-json-rpc-requests.md) |         ✓         |               ✓                |                 |
+
+## MetaMask SDK
+
+The MetaMask SDK ensures a reliable and secure connection to MetaMask Mobile across various platforms. For web dapps, it also enables users to connect via the MetaMask browser extension. The MetaMask SDK can be seamlessly integrated into existing projects without modifying underlying Wallet API calls.
+
+Key features include:
+
+- **Multi-platform support** – Connect from web, desktop, mobile, and gaming platforms.
+- **Seamless mobile integration** – Leverage QR codes or deeplinks to establish persistent connections with MetaMask Mobile.
```

Comment:
```suggestion
- **Seamless mobile integration** – Use QR codes or deeplinks to establish persistent connections with MetaMask Mobile.
```

We try to use "use" instead of "leverage": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/leverage